tensorflow/tools/ci_build/update_version.sh

```diff
@@ -92,6 +93,16 @@ check_existence file "${README_MD}"
 
 sed -i -r -e "s/${OLD_MAJOR}\.${OLD_MINOR}\.([[:alnum:]]+)-/${MAJOR}.${MINOR}.${PIP_PATCH}-/g" "${README_MD}"
 
+# Update the install md files
+NEW_PIP_TAG=$MAJOR.$MINOR.$PIP_PATCH
+OLD_PIP_TAG=$OLD_MAJOR.$OLD_MINOR.$OLD_PIP_PATCH
+
+for file in ${TF_SRC_DIR}/docs_src/install/install_{linux,mac,windows,sources}.md
```

Comment:
Yup, since this was the first release for some of those docs (and the second release for the Java and C APIs in general), they were manual. But going forward, yes, they should be updated concurrently with everything else (the release artifacts are built together).

Thanks for checking!